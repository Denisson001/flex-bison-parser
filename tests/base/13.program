{
    read(a);
    read(a_s);

    b = 5 - a;
    b_s = "abacaba";

    if (a_s == b_s) {
        if (a + b == 5) {
            print("1");
        }
    } else {
        a_s[0] = b_s[1];
    }

    while (k < a && b < a) {
        k = k + 1;
        b = b + 1;
    }

    a = len(a_s) / 2;
    a_s = (a_s + a_s + a_s)[5:5 + a];

    a_s = a_s[0:len(a_s) - 1][0:len(a_s) - 1][0:len(a_s) - 1];

    print(a);
    print(a_s);
}