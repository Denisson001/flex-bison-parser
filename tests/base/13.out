1
3
baab
